src/System.Private.ServiceModel/tools/scripts/BuildCertUtil.cmd

```diff
@@ -37,5 +37,4 @@ echo.
 findstr /ir /c:".*Warning(s)" /c:".*Error(s)" /c:"Time Elapsed.*" "%_buildlog%"
 echo Build Exit Code = %BUILDERRORLEVEL%
 
-endlocal
```

Comment:
Question only: How does removing the `endlocal` impact `call`ing of a batch file from another? We don't `call BuildCertUtil.cmd` today, but will it produce weird environments if we omit `endlocal`?